tensorflow/python/framework/sparse_tensor.py

```diff
@@ -195,6 +195,11 @@ def shape(self):
     """
     return tensor_util.constant_value_as_shape(self._dense_shape)
 
+  @property
+  def name(self):
+    """Get the name of the sparse tensor"""
+    return self.__str__()
```

Comment:
This does not seem correct. Unless we add a name parameter to SparseTensor this does not seems like a reasonable value of `name`.

Reply:
Add the **name** parameter to the SparseTensor. However, I am not sure about the default value of the **name** which should we set. Do you have any ideas?

Reply:
Here I still use **self.__str__()** as the default name. In case that there is no name set in the tensor initialization.